website/app/components/doc/page/header/search/parts/htmlTemplatesItemBody.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { parseAlgoliaHitHighlight } from '@algolia/autocomplete-preset-algolia';
+
+const htmlItemContentTitle = ({ html, title }) => html`
+  <div class="aa-ItemContentTitle">${title}</div>
+`;
+
+const htmlItemContentDescription = ({ html, description }) => html`
+  <div class="aa-ItemContentDescription">${description}</div>
+`;
+
+const snippetInitialText = (text) => {
```

Comment:
we use this trick to return only the first 5 words of a "snippet part" (the non highlighted one) so that the highlighted content is always visible in the description field of the results in the modal panel.

Reply:
Doesn't it return the last five words vs. first five?

Reply:
the last 5 words of the _first_ snippet part (it's broken in multiple parts)